src/components/card/Card.tsx

```diff
@@ -0,0 +1,73 @@
+import SmartLink from '@components/link/SmartLink'
+import { Action } from 'types/card-types'
+import { FC, ReactNode } from 'react'
+
+type Props = {
+  children?: ReactNode
+  header?: ReactNode // will be either a StaticImage or GatsbyImage
+  title?: string
+  subtitle?: string
+  additionalHeaderContent?: ReactNode | ReactNode[]
+  body?: ReactNode
+  actions?: Action | Action[]
+  transparentBorder?: boolean
+  transparentBody?: boolean
+}
+
+const Card: FC<Props> = (props) => {
+  const {
+    children,
+    header,
+    title,
+    subtitle,
+    additionalHeaderContent,
+    body,
+    actions,
+    transparentBorder = false,
+    transparentBody = false,
+  } = props
+
+  // creates the SmartLink component/s for action/s
+  const createActionLinks = (
+    actions: Action | Action[],
```

Comment:
**nitpick:** the property is plural, but allows a single action. I'd let it only handle lists which also removes the need to check for the type below.

Reply:
I've changed the property to be a list only.